toxcore/LAN_discovery.c

```diff
@@ -160,12 +174,12 @@ static void fetch_broadcast_info(uint16_t port)
 
     for (int i = 0; i < n; i++) {
         /* there are interfaces with are incapable of broadcast */
-        if (ioctl(sock, SIOCGIFBRDADDR, &i_faces[i]) < 0) {
+        if (ioctl(sock.socket, SIOCGIFBRDADDR, &i_faces[i]) < 0) {
             continue;
         }
 
-        /* moot check: only TOX_AF_INET returned (backwards compat.) */
-        if (i_faces[i].ifr_broadaddr.sa_family != TOX_AF_INET) {
+        /* moot check: only AF_INET returned (backwards compat.) */
+        if (i_faces[i].ifr_broadaddr.sa_family != AF_INET) {
```

Comment:
TODO avoid using `AF_INET` directly and remove windows headers?

Reply:
Can't really avoid these, because the whole function is heavily system-dependent. Maybe it needs to be moved into network.c.